code/game/turfs/change_turf.dm

```diff
@@ -14,34 +14,33 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 		qdel(thing, force=TRUE)
 
 	if(turf_type)
-		var/turf/newT = ChangeTurf(turf_type, baseturf_type, flags)
-		SSair.remove_from_active(newT)
-		CALCULATE_ADJACENT_TURFS(newT, KILL_EXCITED)
-
-/turf/proc/copyTurf(turf/T)
-	if(T.type != type)
-		T.ChangeTurf(type)
-	if(T.icon_state != icon_state)
-		T.icon_state = icon_state
-	if(T.icon != icon)
-		T.icon = icon
+		var/turf/new_turf = ChangeTurf(turf_type, baseturf_type, flags)
+		SSair.remove_from_active(new_turf)
+		CALCULATE_ADJACENT_TURFS(new_turf, KILL_EXCITED)
+
+/turf/proc/copyTurf(turf/copy_to_turf)
+	if(copy_to_turf.type != type)
+		copy_to_turf.ChangeTurf(type)
+	if(copy_to_turf.icon_state != icon_state)
+		copy_to_turf.icon_state = icon_state
+	if(copy_to_turf.icon != icon)
+		copy_to_turf.icon = icon
 	if(color)
-		T.atom_colours = atom_colours.Copy()
-		T.update_atom_colour()
-	if(T.dir != dir)
-		T.setDir(dir)
-	return T
+		copy_to_turf.atom_colours = atom_colours.Copy()
+		copy_to_turf.update_atom_colour()
+	if(copy_to_turf.dir != dir)
+		copy_to_turf.setDir(dir)
+	return copy_to_turf
 
-/turf/open/copyTurf(turf/T, copy_air = FALSE)
+/turf/open/copyTurf(turf/open/copy_to_turf, copy_air = FALSE)
 	. = ..()
-	if (isopenturf(T))
+	if (isopenturf(copy_to_turf))
```

Comment:
```suggestion
	if (istype(copy_to_turf))
```

Reply:
i'm just going to remove this line, it's supposed to pass an open turf regardless

Reply:
what does "supposed" to mean here? if it's an error state then you should replace with `ASSERT(istype(copy_to_turf))` rather than just removing it altogether

Reply:
the few times this proc is called is all on turf/open, and it's a specific proc for turf/open as well. it's a redundant check

Reply:
![image](https://user-images.githubusercontent.com/35135081/209416061-d67a3817-cb6c-487e-8eb3-7dc93a390ee0.png)